src/search_options.hpp

```diff
@@ -91,14 +90,14 @@ struct LambdaOptions : public SharedOptions
     // scores
     int             gapOpen         = -11;
     int             gapExtend       = -1;
-    int             match           = 0; // only for manual
-    int             misMatch        = 0; // only for manual
+    int             match           = 2; // only for manual
+    int             misMatch        = -3; // only for manual
 
-    int             xDropOff    = 0;
-    int             band        = -1;
-    double          eCutOff     = 0;
+    int             xDropOff    = 30;
+    int             band        = -3;
+    double          eCutOff     = 1e-04;
     int             idCutOff    = 0;
-    unsigned long   maxMatches  = 500;
+    unsigned long   maxMatches  = 256;
```

Comment:
Can you change these types to fixed size integers while you are at it? The ints to `int32_t` and the unsigned long to `uint64_t`.